ros.ws/src/gauge_net/gauge_net/gauge_net/gauge_reader_lite.py

```
@@ -0,0 +1,258 @@
+
+        # cv_bridge for image conversion.
+        self._bridge = CvBridge()
+
+        self.get_logger().info('GaugeReader Node Started')
+
+
+    def detect(self, cv_image):
+        _, buf = cv2.imencode('.jpg', cv_image)
+        b64 = base64.b64encode(buf).decode('utf-8')
+        headers = {'Authorization': f'Bearer {self.token}'}
+        r = requests.post(self.model_server_url + "/detect", json={'image': b64}, headers=headers)
```

Ruff (0.14.2)

Probable use of `requests` call without timeout

(S113)